webpack/webpack.config.cjs

```diff
@@ -236,16 +107,11 @@ module.exports = function(env, argv) {
         {
           test: /\.ts$/,
           loader: 'ts-loader',
-          options: {
-            compilerOptions: dev ? { declaration: true, declarationMap: true } : {},
-          },
+          options: {},
```

Comment:
Is this needed?

Reply:
According to the doc from `webpack-merge`, it seems no need.
But I didn't test it.

Reply:
Tested, working.
So delete.